lib/src/components/NxSearchDropdown/__tests__/NxSearchDropdown.test.tsx

```diff
@@ -232,60 +210,48 @@ describe('NxSearchDropdown', function() {
           { id: '1', displayName: 'One' },
```

Comment:
I can't comment that line, but the name of this test should be adjusted.  The load wrapper is an impl detail.  How about 'renders buttons with type "button" for each match'

Reply:
> I can't comment that line, but the name of this test should be adjusted. The load wrapper is an impl detail. How about 'renders buttons with type "button" for each match'

Should be changed here: https://github.com/sonatype/sonatype-react-shared-components/pull/933/files#diff-8faf831d9f7a4427b8d4674137223bd7eb914e2b245eddc1308562fa0ce5a3f9R212-R218